config/sync/core.extension.yml

```diff
@@ -83,7 +84,6 @@ module:
   nidirect_hospital_waiting_times: 0
   nidirect_money_advice_articles: 0
   nidirect_news: 0
-  nidirect_webforms: 0
```

Comment:
Is this module removal expected?

Reply:
Hell no, and it's enabled on my site... odd.